libmscore/rest.cpp

```diff
@@ -786,7 +786,7 @@ void Rest::reset()
 
 qreal Rest::mag() const
       {
-      qreal m = staff()->mag(this);
+      qreal m = staff() ? staff()->mag(this) : 1.0;
```

Comment:
not really needed, but in sync with a change in https://github.com/musescore/MuseScore/pull/6365/files#diff-14e088070bf5809a99311618482e148dL640-R644, a PR for master